extensions/wikia/ContributionPrototype/CPEditor.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace ContributionPrototype;
+
+use Article;
+use Http;
+use MWHttpRequest;
+use EditPage;
+use OutputPage;
+use Wikia;
+use Wikia\Logger\Loggable;
+use Wikia\Service\Gateway\UrlProvider;
+
+class CPEditor extends EditPage {
+
+	use Loggable;
+
+	function __construct(Article $article) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method "__construct" to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)